### ISSUE TITLE
November MB Release

### ISSUE THREAD

**Issue by beccamc** [dotnet/machinelearning-modelbuilder#1911]
Commit:  https://github.com/dotnet/machinelearning-tools/commit/4d615ba420977a1685c11b97f9fc3acbdd84b5c5
Release notes: 

**2022** Version number: 16.9.1.2205404
**2022** Build: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5588797
**2019** Version number: 16.9.1.2205403 
**2019** Build: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=5588184

Tracking work item: https://github.com/dotnet/machinelearning-modelbuilder/issues/1707

## Testing
- [x] Test Local Scenarios
- [x] Test Azure Scenarios
- [x] Test Generated Code
- [x] Test VS 2019
- [x] Test VS 2022
- [x] Test GPU

**Comment by beccamc** [comment]
With the updates of #1905 Model Builder now generates: 

.NET 6 for VS 2022
.NET 5 for VS 2019.  

We need to update the docs. (and confirm with team that above is true 😀)

**Comment by beccamc** [comment]
Blocked on this issue: https://github.com/dotnet/machinelearning-modelbuilder/issues/1938

**Comment by beccamc** [comment]
Tested that it won't install for 2019 versions earlier than 16.11. In this screenshot, it only shows install option for version 16.11, not for version 16.9

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/61f9e267-4a96-4b52-b484-708f8691146d)

**Comment by beccamc** [comment]
^ Above no longer relevant. Have rolled back the SDK problem in PR https://github.com/dotnet/machinelearning-tools/pull/1370

**Comment by beccamc** [comment]
Complete test pass by Qin: onenote:https://microsoft.sharepoint.com/teams/CloudPlatformToolsDevOpsCTI/Shared%20Documents/Team%20Notes/DevOps%20CTI%20Wuxi/12%20Model%20Builder/Test%20Result.one#16.9.1.2161502_SanityTestPassMLNet.CLITestPass&section-id=%7B73026020-B53E-45B5-97E9-49E94EE8CB12%7D&page-id=%7B3970082A-6B0D-4D16-8166-CE593BA46873%7D&end
